backend/GymBeam/Controllers/AuthenticationController.cs

```diff
@@ -66,7 +66,7 @@ public async Task<ActionResult<UserResponse>> GoogleCallback([FromQuery] string
                     Username = userInfo.Email
                 });
 
-                return await Login(new LoginRequest
+                await Login(new LoginRequest
```

Comment:
If  we doesn't return user response anymore we should modify return type of this function, right?

Reply:
Done